sites/fast-component-explorer/app/explorer.tsx

```diff
@@ -189,165 +197,142 @@ class Explorer extends Foundation<
         }
 
         return (
-            <Background value={neutralLayerL1}>
-                <Container>
-                    <Row style={{ flex: "1" }}>
-                        <Pane width={260}>
+            <div className={this.getContainerClassNames()}>
+                <div className={"pane pane__start"}>
+                    <Logo
+                        className={"logo"}
+                        logo={FASTInlineLogo}
+                        title={"Component Explorer"}
+                    />
+                    <NavigationMenu
+                        menu={menu}
+                        expanded={true}
+                        activeLocation={this.state.locationPathname}
+                        onLocationUpdate={this.handleUpdateRoute}
+                    />
+                </div>
+                <div className={"canvas"}>
+                    <div
+                        className={`canvas-overlay${
+                            this.state.mobileFormVisible ||
```

Comment:
[classNames](https://www.npmjs.com/package/@microsoft/fast-web-utilities#classnames) can simplify all these class name concatenation expressions; I see a couple of them around.

Reply:
Forgot about that utility, it's been a minute!